tools/config-generator/udpaterelease.go

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	maxReleaseBranches = 4
+	// Assuming all repo with 0.14 branch also has 0.15 branch, this will be
+	// derived from the branch in following PR
+	latest = "0.15"
+)
+
+func upgradeReleaseBranchesTemplate(name string) error {
```

Comment:
Nit: I'd be more specific with the `name` argument name. `configFileName` might be more concrete on the content and purpose